src/workflows/ingestion/bd_rhapsody/test.nf

```diff
@@ -32,9 +33,20 @@ workflow test_wf {
       assert data.output_h5mu.toString().endsWith(".h5mu") : "Output file should be a h5mu file. Found: ${output[1]}"
       "Output: $output"
     }
+
+    | bd_rhapsody_test.run(
+      fromState: { output ->
+        [
+          input: output[1].output_h5mu
+        ]
+      }
+    )
```

Comment:
This can be shortened to
```suggestion
    | bd_rhapsody_test.run(
      fromState: ["input": "output"]
    )
```

Reply:
I think i had that before but then I'm getting: `Error in module 'bd_rhapsody_test' id 'foo': required input argument 'input' is missing.`

Reply:
My mistake, should be 
```suggestion
    | bd_rhapsody_test.run(
      fromState: ["input": "output_h5mu"]
    )
```